### PR TITLE
feature PR ready

### PR DESCRIPTION
**Pull request by pananimals**

### CODE REVIEW

File: src/js/langData.js [hunk 1]
```diff
@@ -89,6 +90,7 @@ export const arrLang = {
     'menu-production': 'Production',
     'menu-about': 'About us',
     'menu-contact': 'Contact',
+    'menu-rodo': 'Rodo',
```
Comment (michalkowol) [inline src/js/langData.js]: - [ ] Rodo = GDPR

File: src/js/langData.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ export const arrLang = {
     'menu-production': 'Produkcja',
     'menu-about': 'O firmie',
     'menu-contact': 'Kontakt',
+    'menu-rodo': 'Rodo',
```
Comment (michalkowol) [inline src/js/langData.js]: - [ ] Please change it to uppercase

Comment (pananimals) [reply src/js/langData.js]: Ok, but I thought we were trying to avoid uppercases

Comment (michalkowol) [reply src/js/langData.js]: Yes, but in this case it is the name

**Pull request merged**